LibertyDSNP/example-client
# updated regex for youtube videos

Purpose
---------------
bug
[https://www.pivotaltracker.com/story/show/179914342](https://www.pivotaltracker.com/story/show/179914342)
maybe a character got deleted in the regex or something but isYoutubeVideo is returning null when it should not be.

Solution
---------------
- make the regex correct so we can see the thumbnail.


Change summary
---------------
* Changed regex to something more simple

Steps to Verify
----------------
1. create new post with youtube video and see thumbnail.


Additional details / screenshot
----------------
working!
![Uploading Screen Shot 2021-10-12 at 4.05.52 PM.png…]()
